cms/templates/js/active-video-upload-list.underscore

```diff
@@ -6,5 +6,5 @@
 </form>
 <section class="active-video-upload-container">
     <h3 class="sr"><%= gettext("Active Uploads") %></h3>
-    <ul class="active-video-upload-list"></ul>
+    <ul class="active-video-upload-list" aria-live="polite"></ul>
```

Comment:
@pomegranited If you put a role=alert on that container, anything you insert into the region should be read (even by voiceover, I tested). And even if it is subsequently removed.  Whether it stays on screen or not shouldn't matter.  This is why I recommended adding the successful message after the status message, even if its displayed on screen so quickly its never seen by a visual user.   Leaving messages onscreen should yield much more predictable results.  However, how would this look if the user uploads 30 videos at once.  In that scenario, it might actually be beneficial to see all the cards, that way the user could easily identify any uploads that failed.  Perhaps there could be a button to clear notifications? (although, this PR is to remove a button...)

NOTE: I haven't tested this out with multiple video uploads.  Hopefully, each notice/status change does not interrupt any notification which is currently being announced.